common/engine/keyboardprocessor/tests/unit/kmx/042 - long context and split deadkeys.kmn

```diff
@@ -0,0 +1,12 @@
+c Description: Tests that we don't split a dk in context
+c keys: [K_SPACE]
+c context: \u00ffff\u000008\u00000001abcdefghijklmnopqrstuvwxyzabcdefghijklmnopqrstuvwxyzabcdefgh
+c expected: \u00ffff\u000008\u00000001abcdefghijklmnopqrstuvwxyzabcdefghijklmnopqrstuvwxyzabcdefgh\u0020
+
+store(&version) '6.0'
+
+begin Unicode > use(Main)
+
+group(Main) using keys
+
```

Comment:
```suggestion

```suggestion

c Test 1: test that we haven't split a deadkey code: `UC_SENTINEL CODE_DEADKEY <deadkey_number>`
c   here, looking at last char only: `deadkey_number` = `U+0001`
```